pages/_app.jsx

```diff
@@ -27,7 +27,7 @@ import 'assets/styles/themes.css'
 import 'assets/styles/bnc-onboard--custom.css'
 import 'assets/styles/reach--custom.css'
 
-const queryCache = new QueryCache()
+export const queryClient = new QueryClient()
```

Comment:
Curious, why the `export` ?

Reply:
I guess it doesn't need to be, I just copied from the other repo